src/main/java/com/rcv/GuiConfigController.java

```diff
@@ -156,7 +156,7 @@
 
   public void buttonNewConfigClicked() {
     if (checkForSaveAndContinue()) {
-      Logger.log(Level.INFO, "Creating new config.");
+      Logger.log(Level.INFO, "Creating new contest.");
```

Comment:
Hmm, don't agree with this change. I could settle for "contest config" though. I suggest reverting these three "config" => "contest" changes and filing an issue so we can treat this more comprehensively if you think it's important. The problem is that now we have these three references changed, but there are cases where "config" is used in messages communicated to the user, title bars, etc.

Reply:
Sure I can change them back.  Generally I think we should avoid using terms like "config" with the user which are only used by programmers but it doesn't need to happen now.